spark/spark-spa/src/main/resources/velocity/layouts/component_layout.vm

```diff
@@ -26,18 +26,16 @@
   <title>${viewTitle} - Go</title>
 
   <link rel="shortcut icon" href="/go/${railsAssetsService.getAssetPath('cruise.ico')}"/>
-    #if(${railsAssetsService.getAssetPath("single_page_apps/${controllerName}.css")} != "")
-      <link href="/go/${railsAssetsService.getAssetPath("single_page_apps/${controllerName}.css")}" media="screen"
-            rel="stylesheet"/>
-    #end
+#if(${railsAssetsService.getAssetPath("single_page_apps/${controllerName}.css")})
```

Comment:
For this line to work, `single_page_apps/kitchen_sink.scss` is required. Can we add an empty file for this to work?